method_comparison/app.py

```diff
@@ -222,7 +222,7 @@ def build_app(df):
 
         with gr.Row():
             x_default = (
-                "cuda_memory_max" if "cuda_memory_max" in metric_preferences else list(metric_preferences.keys())[0]
+                "accelerator_memory_max" if "accelerator_memory_max" in metric_preferences else list(metric_preferences.keys())[0]
```

Comment:
Could you please add line breaks (or run `ruff format app.py`)?

Reply:
done, and done `ruff format` for other 2 python files, thx.